specify_pose/proposal.md

```diff
@@ -321,19 +288,17 @@ in code, the order of operations, etc.).
 
 #### 1.2 Conversion to SDFormat 1.9
 
-When SDFormat files are converted from SDFormat <=1.8 to 1.9, the `//pose` tags
```

Comment:
I removed this section because I think this was a remnant of when this document proposed to deprecate RPY with radians. IMO, we should leave poses in older files intact. This will keep converted from getting verbose. Also, we currently print out only attributes with non-default values. We can have a special case for `//pose` so that `//pose/rotation_format="euler_rpy"` and `//pose/@degrees=false` get printed even though they are the default values, but the added complexity doesn't seem justified.